src/Workspaces/Core/Portable/Shared/Extensions/DocumentExtensions.cs

```diff
@@ -49,6 +49,6 @@ public static async Task<(Document document, SemanticModel? semanticModel)> GetP
         }
 
         internal static Document WithSolutionOptions(this Document document, OptionSet options)
```

Comment:
what happens with this method in the future?  no longer needed?

Reply:
Yes, I think we'll be able to remove.